pkg/publicapi/apimodels/job.go

```diff
@@ -107,7 +107,6 @@ type ListJobHistoryRequest struct {
 	Since       int64  `query:"since" validate:"min=0"`
 	EventType   string `query:"event_type" validate:"omitempty,oneof=all job execution"`
 	ExecutionID string `query:"execution_id" validate:"omitempty"`
-	NodeID      string `query:"node_id" validate:"omitempty"`
```

Comment:
Would be great to get rid of it here too https://github.com/bacalhau-project/bacalhau/blob/main/pkg/publicapi/endpoint/orchestrator/job.go#L275